### PR TITLE
Js sdk docs

### PR DESCRIPTION
**Pull request by Caperious**
Added documentation for js-sdk

### CODE REVIEW

File: source/includes/_filters.md [hunk 1]
```diff
@@ -76,7 +145,146 @@ curl "...?filter[where][number][like]=2018.*&filter[where][number][options]=i"
...
+      }
+    }
+  }
+}
+// LIKE | NLIKE / options
+var queryParams = {
+  filter: {
+    where: {
+      number: {
+        number: {
```
Comment (BostjanPisler) [inline source/includes/_filters.md]: 🧐

File: source/includes/_filters.md [hunk 2]
```diff
@@ -76,7 +145,146 @@ curl "...?filter[where][number][like]=2018.*&filter[where][number][options]=i"
...
+      }
+    }
+  }
+}
+// or
+var queryParams = {
+  filter: {
+    where: {
+      number: {
+        number: {
```
Comment (BostjanPisler) [inline source/includes/_filters.md]: 🙈

File: source/includes/_filters.md [hunk 3]
```diff
@@ -76,7 +145,146 @@ curl "...?filter[where][number][like]=2018.*&filter[where][number][options]=i"
...
+      }
+    }
+  }
+}
+// regExp
+var queryParams = {
+  filter: {
+    where: {
+      number: {
+        number: {
```
Comment (BostjanPisler) [inline source/includes/_filters.md]: 😭

File: source/includes/_documents.md [hunk 4]
```diff
@@ -16,7 +16,28 @@ curl "https://api.spaceinvoices.com/v1/organizations/:id/documents" \
   -d _documentItems[0][unit]="Item" \
   -d _documentItems[0][price]="1000"
 ```
-
+```javascript
+spaceInvoices.documents.create(organisation.id, {
```
Comment (BostjanPisler) [inline source/includes/_documents.md]: organization "Z"

Comment (Caperious) [reply source/includes/_documents.md]: Done

**Pull request merged**